back/src/Domain/Buying/Buying.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Domain\Buying;
+
+use App\Domain\Common\Behavior\TimestampableTrait;
+use App\Domain\Common\Behavior\UlidTrait;
+use App\Domain\Apiary\Apiary;
+use Symfony\Component\DependencyInjection\Attribute\Exclude;
+
+#[Exclude]
+class Buying
+{
+    use UlidTrait;
+    use TimestampableTrait;
+
+    private string $label;
+    private float $price;
+    private Apiary $apiary;
```

Comment:
Hmmm avec le recul, il serait intéressant d'avoir ce champ nullable, au cas où l'utilisateur voudra juste enregistrer un achat qui n'est pas intrinsèquement lié à un rucher. Cf. Buying.orm.xml (bien penser à le préciser dans le schéma 😬 )
```suggestion
    private ?Apiary $apiary = null;
```